anda/lib/process-cpp/process-cpp.spec

```diff
@@ -0,0 +1,84 @@
+%global forgeurl https://gitlab.com/ubports/development/core/lib-cpp/process-cpp
+%global commit ee6d99a3278343f5fdcec7ed3dad38763e257310
+%forgemeta
+
+Name:          process-cpp
+Version:       3.0.2
```

Comment:
`3.0.2` is unreleased. I'm not sure if we should do it like this… Is is a must to use the latest commit?

Reply:
My bad it looked like `3.0.2` was released https://gitlab.com/ubports/development/core/lib-cpp/process-cpp/-/blob/main/debian/changelog

Reply:
I mean it does say
```
process-cpp (3.0.2) UNRELEASED; urgency=medium
```
unreleased in caps right there… so…